ext/bg/js/options.js

```diff
@@ -205,6 +260,24 @@ function optionsVersion(options) {
             } else {
                 options.scanning.modifier = 'none';
             }
+        },
+        () => {
+            if (options.general.groupResults) {
+                options.general.resultOutputMode = 'group';
+            } else {
+                options.general.resultOutputMode = 'split';
+            }
+            options.general.compactTags = false;
+            options.general.compactGlossaries = false;
+            if (utilStringHashCode(options.anki.fieldTemplates) !== -805327496) { // a3c8508031a1073629803d0616a2ee416cd3cccc
+                options.anki.fieldTemplates = '{{#if merge}}\n' +
```

Comment:
Prefer template string as it does not require explicit line breaks and you can just use all of the expressions inline without having to combine strings with `+` operator.